### PR TITLE
Upgrade to OpenSearch and Dashboards 2.13.0

### PR DESCRIPTION
**Pull request by gaiksaya**
### Description
Upgrade to OpenSearch and Dashboards 2.13.0
 
### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/4433
 
### Check List
- [x] Commits are signed per the DCO using --signoff

For any changes to files within Helm chart directories:
- [x] Helm chart version bumped
- [x] Helm chart `CHANGELOG.md` updated to reflect change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/helm-charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: charts/opensearch-dashboards/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [2.17.0]
```
Comment (prudhvigodithi) [inline charts/opensearch-dashboards/CHANGELOG.md:16]: Hey @gaiksaya can you update the change log at bottom of this file?

File: charts/opensearch/CHANGELOG.md [hunk 2]
```diff
@@ -12,9 +12,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 ### Fixed
 ### Security
-
 ---
-
+## [2.19.0]
```
Comment (prudhvigodithi) [inline charts/opensearch/CHANGELOG.md:16]: Same can you update the change log at bottom of this file?

File: charts/opensearch-dashboards/CHANGELOG.md (hunk 1 above)
Comment (peterzhuamazon) [reply charts/opensearch-dashboards/CHANGELOG.md:16]: ^^

Comment (gaiksaya) [reply charts/opensearch-dashboards/CHANGELOG.md:16]: Done

File: charts/opensearch/CHANGELOG.md (hunk 2 above)
Comment (gaiksaya) [reply charts/opensearch/CHANGELOG.md:16]: Done

**Review by peterzhuamazon — APPROVED** [review]

File: charts/opensearch-dashboards/CHANGELOG.md (hunk 1 above)
Comment (prudhvigodithi) [reply charts/opensearch-dashboards/CHANGELOG.md:16]: Looks like [2.16.1] was missed in past PR.

**Review by prudhvigodithi — APPROVED** [review]

**Comment by adit-cmd** [comment]
Hi All,
Any reason why the latest version in git is changed back to 1.19.0?

**Comment by peterzhuamazon** [comment]
> Hi All, Any reason why the latest version in git is changed back to 1.19.0?

* https://github.com/opensearch-project/helm-charts/pull/540

**Comment by peterzhuamazon** [comment]
Seems like we need to fix this with 1.6.0 version of action.
* https://github.com/helm/chart-releaser-action/releases/tag/v1.6.0

**Pull request merged**